bigchaindb/common/schema/__init__.py

```diff
@@ -29,6 +29,8 @@ def _load_schema(name):
 
 
 TX_SCHEMA_PATH, TX_SCHEMA = _load_schema('transaction')
```

Comment:
Should we rename this schema from `transaction` to something like `transaction_common` so that it is easier to understand what is going on?